warehouse/ingest-scripts/src/main/resources/bin/ingest/start-loader.sh

```diff
@@ -83,15 +83,15 @@ if [[${TOTAL} >0 ]]; then
     COUNT=${NUM_MAP_LOADERS_COPY[$LOADER]}
     echo "starting $COUNT map file loaders for ${MAP_LOADER_HDFS_NAME_NODES[$LOADER]} ..."
     SHUTDOWN_PORT=24100
-    portInUse=$(sudo netstat -tupln | grep $SHUTDOWN_PORT)
+    portInUse=$(ps -eaf | grep [b]ulkIngestMap | grep $SHUTDOWN_PORT)
```

Comment:
Rather than `ps -eaf | grep [b]ulkIngestMap | grep $SHUTDOWN_PORT`, would it be better to use something like `lsof -i :$SHUTDOWN_PORT` instead, so at least all the current user's processes can be queried?

Reply:
Both could work - Since the process we're looking to grab is an ingest process we may not necessarily need to bring in anything more processes than that in order to verify the loaders aren't using the port in question. I'll test this out a little and adjust if needed.